src/btwa_api/app/api/endpoints.py

```diff
@@ -1,44 +1,70 @@
+from typing import List
+
 from fastapi import APIRouter, Depends, HTTPException
 from fastapi.security import HTTPBasicCredentials
+from pydantic import BaseModel
 
-from ..db.database import Database
+from .utils import get_or_404
+from ..db.firebase.firebase import Firebase, get_firebase
+from ..db.sql.database import Database
 from ..db.utils import get_db
 from ..security import check_handler_auth, security
-from starlette.responses import JSONResponse
 
 router = APIRouter()
 
 
-@router.get("/register/{guid}")
-def register(guid: str,
+class User(BaseModel):
+    fuid: str
+    status: str
+
+
+@router.get("/get-user/{phone}", response_model=User)
+def get_user(phone: str,
              credentials: HTTPBasicCredentials = Depends(security),
+             firebase: Firebase = Depends(get_firebase),
              db: Database = Depends(get_db)):
+    """Find user by his phone number."""
+    check_handler_auth(db, credentials)
```

Comment:
Yes, it should be a middleware or a decorator, but there's no point in changing this, because it will probably go away. This PR is just an MVP for initial demonstration.